src/test/java/com/puppycrawl/tools/checkstyle/internal/XpathRegressionTest.java

```diff
@@ -203,13 +205,19 @@ protected String getResourceLocation() {
 
     @Test
     public void validateIncompatibleJavadocCheckNames() throws IOException {
-        final Set<Class<?>> abstractJavadocCheckNames = CheckUtil.getCheckstyleChecks()
+        // subclasses of AbstractJavadocCheck
+        final Stream<Class<?>> abstractJavadocCheckNames = CheckUtil.getCheckstyleChecks()
                 .stream()
-                .filter(AbstractJavadocCheck.class::isAssignableFrom)
+                .filter(AbstractJavadocCheck.class::isAssignableFrom);
+        // hardcoded exceptions, will eventually include JavadocMethodCheck and JavadocTypeCheck
+        final Stream<Class<?>> otherJavadocCheckNames = Stream.of(JavadocStyleCheck.class);
```

Comment:
Do we need 2 streams? Does it make sense to combine them and the assignment of the `Set`?

Reply:
Since the latter stream is being moved to a field, I'll just revert the former stream to `Set` and work on that as it's closer to the original behavior anyway.